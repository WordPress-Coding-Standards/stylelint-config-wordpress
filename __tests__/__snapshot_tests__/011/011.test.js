'use strict';

const stylelint = require( 'stylelint' ),
	systemTestUtils = require( '../systemTestUtils.js' );

it( '011', () => {
	return stylelint.lint({
		files: [systemTestUtils.caseStylesheetGlob( '011' )],
		configFile: systemTestUtils.caseConfig( '011' ),
	}).then( ({ results }) => {
		expect( systemTestUtils.prepResults( results ) ).toMatchSnapshot();
	});
});
