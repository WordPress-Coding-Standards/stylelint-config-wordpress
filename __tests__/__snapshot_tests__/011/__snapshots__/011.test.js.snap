// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`011 1`] = `
Array [
  Object {
    "deprecations": Array [],
    "errored": true,
    "ignored": undefined,
    "invalidOptionWarnings": Array [],
    "parseErrors": Array [],
    "source": "011/stylesheet.css",
    "warnings": Array [
      Object {
        "column": 1,
        "line": 14,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
    ],
  },
]
`;
